modules/gitops-creating-a-new-client-using-keycloak.adoc

```diff
@@ -36,10 +38,37 @@ metadata:
 spec:
   sso:
     provider: keycloak
+    keycloak:
+      rootCA: "<PEM-encoded-root-certificate>" <1>
   server:
     route:
-     enabled: true
+      enabled: true
+----
+<1> A custom certificate used to verify the Keycloak's TLS certificate.
++ 
+The Operator reconciles changes in the `.spec.keycloak.rootCA` field and updates the `oidc.config` in `argocd-cm` configuration map with the PEM encoded root certificate.
+
+[NOTE]
+====
+To proceed with an insecure connection, leave the value of `rootCA` empty and use `skip.verify` as shown below:
```

Comment:
Do not we need to use the complete parameter name, instead of just specifying `skip.verify`?

```suggestion
To proceed with an insecure connection, leave the value of `rootCA` empty and use the `oidc.tls.insecure.skip.verify` parameter as shown below:
```